app/helpers/user_api.rb

```diff
@@ -33,4 +37,12 @@ def self.can_administer?(user_id, competition_id)
     end
     permissions['can_administer_competitions']['scope'] == '*' || permissions['can_administer_competitions']['scope'].include?(competition_id)
   end
+
+  def self.personal_records(user_id)
+    if Rails.env.production?
+      HTTParty.get(permissions_path(user_id), headers: { WCA_API_HEADER => self.get_wca_token })
```

Comment:
Don't think we need a wca_token for this route! Also this still has the permissions_path ;)

Reply:
Ah lol totally forgot to change that line, fixed